app/controllers/api/base_controller.rb

```diff
@@ -225,6 +226,15 @@ def paginate_options
       }
     end
 
+    def assign_lone_taxonomies
+      [Location, Organization].each do |taxonomy|
+        next if params["#{taxonomy.to_s.downcase}_ids"]
+        next unless resource_class.reflections.has_key? taxonomy.to_s.downcase.pluralize
```

Comment:
Nice, does this also apply for objects that may only belong to a single taxonomy?
E.g. a host can only be in one taxonomy. If there is only one taxonomy defined, we can assign it as well. Unless we already have a different mechanism for this somewhere else.

Reply:
It does not currently. However, one common way of host creation via api is using fact import, which does set the default taxonomy (even if there is more than one). If you think it is needed we can add that in a follow-up PR.

Reply:
These questions were of rhetorical nature. ;-)
I think we should do it for Objects with a single taxonomy as well - at least for consistency's sake.

Reply:
you know what, i'll fix it in this one